uvm/servlets/admin/config/administration/MainModel.js

```diff
@@ -9,18 +9,18 @@ Ext.define('Ung.config.administration.MainModel', {
 
         adminSettings: null,
         systemSettings: null,
-        skinSettings: null,
+        //skinSettings: null,
 
         serverCertificates: null,
         rootCertificateInformation: null,
         serverCertificateVerification: null,
-        skinsList: null,
+        //skinsList: null,
```

Comment:
Let's just delete it, not comment it out.

Reply:
removed